### PR TITLE
Introduces a new "governance-modules" package

### LINKED ISSUE #869: Move election module to a new governance package, and make election module non-generic

### PR DESCRIPTION
**Pull request by ajsantander**
Fix #869

### CODE REVIEW

File: packages/governance-modules/test/contracts/modules/ElectionModule/Dismiss.test.js [hunk 1]
```diff
@@ -62,6 +62,11 @@ describe('ElectionModule (dismiss)', () => {
   });
 
   describe('when the module is initialized', function () {
+    before('deploy debt shares mock', async function () {
+      const factory = await ethers.getContractFactory('DebtSareMock');
```
Comment (leomassazza) [inline packages/governance-modules/test/contracts/modules/ElectionModule/Dismiss.test.js:66]: Typo: DebtShareMock

File: packages/governance-modules/test/contracts/modules/ElectionModule/Elect.test.js [hunk 2]
```diff
@@ -36,6 +36,11 @@ describe('ElectionModule (vote)', () => {
   });
 
   describe('when the module is initialized', function () {
+    before('deploy debt shares mock', async function () {
+      const factory = await ethers.getContractFactory('DebtSareMock');
```
Comment (leomassazza) [inline packages/governance-modules/test/contracts/modules/ElectionModule/Elect.test.js:40]: Same typo

File: packages/governance-modules/test/contracts/modules/ElectionModule/Evaluate.test.js [hunk 3]
```diff
@@ -39,6 +39,11 @@ describe('ElectionModule (evaluate)', () => {
   });
 
   describe('when the module is initialized', function () {
+    before('deploy debt shares mock', async function () {
+      const factory = await ethers.getContractFactory('DebtSareMock');
```
Comment (leomassazza) [inline packages/governance-modules/test/contracts/modules/ElectionModule/Evaluate.test.js:43]: dito

**Review by leomassazza — APPROVED** [review]
Approved, but there's a typo repeated on several files. find/replaceAll :)

File: packages/core-js/test/utils/merkle-tree/balance-tree.test.js [hunk 4]
```diff
@@ -23,7 +23,7 @@ describe('utils/merkle-tree/balance-tree.js', function () {
       const proof = tree.getProof(balances[0].account, balances[0].amount);
       const root = tree.getHexRoot();
 
-      equal(proof.length > 2, true);
+      equal(proof.length >= 2, true);
```
Comment (leomassazza) [inline packages/core-js/test/utils/merkle-tree/balance-tree.test.js:26]: Added this fix on the fly to resolve the flacky test

File: packages/governance-modules/test/contracts/modules/ElectionModule/Dismiss.test.js (hunk 1 above)
Comment (ajsantander) [reply packages/governance-modules/test/contracts/modules/ElectionModule/Dismiss.test.js]: Fixed!

**Pull request merged**